initrd.txt

```diff
@@ -0,0 +1,1397 @@
+[   12539904]  ./rootfs-0/kernel/x86/microcode/GenuineIntel.bin
```

Comment:
The microcode updates can be delayed by defining a service that does `echo 1 > /sys/devices/system/cpu/microcode/reload` once we would symlink/copy them to `/lib/firmware/{intel-ucode,amd-ucode}` in the initrd from `/sysusr`.